app/lib/middleware/localAPI.js

```diff
@@ -0,0 +1,141 @@
+const _ = require('lodash')
```

Comment:
Question about this entire file—is this used only for development purposes? Do you expect to reuse any of this code when talking to the live API?

Reply:
Correct.  I don't believe there will be any reuse.  It's only for development and test purposes.